docs/BOOT_LOADER_SPECIFICATION.md

```diff
@@ -190,14 +191,14 @@ The following keys are recognized:
 
 * `sort-key` is a short string used for sorting entries on display. This should
   typically be initialized from the `IMAGE_ID=` or `ID=` fields of
-  `/etc/os-release`, possibly with an additional suffix. This field is
-  optional. If set, it is used as primary sorting key for the entries on
-  display (lexicographically increasing). It does not have to be unique (and
-  usually is not). If non-unique the the `machine-id` (lexicographically
-  increasing) and `version` (lexicographically decreasing, i.e. newest version
-  first) fields described above are used as secondary/ternary sorting keys. If
-  this field is not set entries are typically sorted by the `.conf` file name
-  of the entry.
+  [os-release](https://www.freedesktop.org/software/systemd/man/os-release.html),
+  possibly with an additional suffix. This field is optional. If set, it is
+  used as primary sorting key for the entries on display (lexicographically
+  increasing). It does not have to be unique (and usually is not). If
+  non-unique the the `machine-id` (lexicographically increasing) and `version`
```

Comment:
Double the.

Reply:
This is also just moving the text around. It is fixed in a later commit.